README.md

```diff
@@ -159,19 +159,20 @@ A request (to read or write) has arrived for a document located at
 
 1. First, it checks for the existence of an individual corresponding ACL
   resource. (That is, it checks if `paper1.acl` exists.) If this individual ACL
-  resource exists, the server uses the Authorization statements in that ACL. No
-  other statements apply.
+  resource exists, the server uses the
+  `acl:accessTo </documents/papers/paper1.acl>` statements in that ACL. No other
+  statements apply.
 2. If no individual ACL exists, the server next checks to see if the
   `/documents/papers/` container (in which the document resides) has its own
-  ACL resource (here, `/documents/papers/.acl`). If it finds that, the server
-  reads each authorization in the container's ACL, and if any of them contain an
-  `acl:default` predicate, the server will use them (as if they were
-  specified in `paper1.acl`). Again, if any such authorizations are found, the
-  process stops there and no other statements apply.
+  ACL resource (here, `/documents/papers/.acl`). If it finds that,
+  the server uses the `acl:default </documents/papers/>` statements
+  in that ACL. No other statements apply. Note that this behavior is likely to
+  change in the future, so that ACL docs that contain no `acl:default` statements
+  are skipped. Please join the discussion at https://github.com/solid/specification/issues/55.
```

Comment:
I think we don't need to mention the discussion, since we have a published intention to rewrite the spec, so it can be understood from the open issues.